js/latoken1.js

```diff
@@ -170,41 +191,61 @@ module.exports = class latoken1 extends Exchange {
-            const symbol = base + '/' + quote;
             const pricePrecisionString = this.safeString (market, 'pricePrecision');
             const priceLimit = this.parsePrecision (pricePrecisionString);
-            const precision = {
-                'price': parseInt (pricePrecisionString),
-                'amount': this.safeInteger (market, 'amountPrecision'),
-            };
-            const limits = {
-                'amount': {
-                    'min': this.safeNumber (market, 'minQty'),
-                    'max': undefined,
-                },
-                'price': {
-                    'min': this.parseNumber (priceLimit),
-                    'max': undefined,
-                },
-                'cost': {
-                    'min': undefined,
-                    'max': undefined,
-                },
-            };
+            const fees = this.safeValue (this.fees, 'trading');
+            const defaultTaker = this.safeNumber (fees, 'taker');
+            const defaultMaker = this.safeNumber (fees, 'maker');
             result.push ({
                 'id': id,
                 'numericId': numericId,
-                'info': market,
-                'symbol': symbol,
+                'symbol': base + '/' + quote,
                 'base': base,
                 'quote': quote,
+                'settle': undefined,
                 'baseId': baseId,
                 'quoteId': quoteId,
+                'settleId': undefined,
                 'type': 'spot',
                 'spot': true,
-                'active': undefined, // assuming true
-                'precision': precision,
-                'limits': limits,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'active': undefined,
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'taker': this.safeNumber (market, 'takerFee', defaultTaker),
+                'maker': this.safeNumber (market, 'makerFee', defaultMaker),
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'precision': {
+                    'price': parseInt (pricePrecisionString),
+                    'amount': this.safeInteger (market, 'amountPrecision'),
+                },
+                'limits': {
+                    'leverage': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': this.safeNumber (market, 'minQty'),
+                        'max': undefined,
+                    },
+                    'price': {
+                        'min': this.parseNumber (priceLimit),
```

Comment:
Let's set it to `undefined`, we should not really derive the limits from precision.